File: Exemplos_desafios_linguagens/java/DESAFIOS-DE-CALCULOS/src/desafioEmJava/desafio.java
package desafioEmJava;

import java.util.Scanner; 

public class desafio {
    public static void main(String[] args) {
        Scanner leitorDeEntradas = new Scanner(System.in);
        System.out.println("Digite o valor do seu salário: ");
        float valorSalario = leitorDeEntradas.nextFloat();
        System.out.println("Digite o valor dos seus benefícios: ");
        float valorBeneficios = leitorDeEntradas.nextFloat();

        float valorImposto = 0;

        if (valorSalario >= 0 && valorSalario <= 1100) {
            valorImposto = 0.05F * valorSalario;
        } else if (valorSalario >= 1100.01 && valorSalario <= 2500) {
            valorImposto = 0.1F * valorSalario;
        } else {
            valorImposto = 0.15F * valorSalario; 
        }
        float saida = valorSalario -valorImposto + valorBeneficios;
        System.out.println(String.format("%.2f", saida));
    }
}

File: Exemplos_desafios_linguagens/python/desafio.py

def calcular_impostos(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 0.005
    elif (salario > 1100 and salario <= 2500):
        aliquota = 0.10
    elif (salario > 2500):
        aliquota = 0.15
    return (salario * aliquota)

valorSalario = float(input("Digite o valor do seu salário: "))
valorBeneficios = float(input("Digite o valor dos seus benefícios: "))

valorImposto = calcular_impostos((valorSalario))

saida = (valorSalario - valorImposto) + valorBeneficios

print(f"Salário líquido: {saida:.2f}")
    